tests/bluetooth/audio/ascs/Kconfig

```diff
@@ -0,0 +1,29 @@
+# Bluetooth Host configuration options for ASCS Unit Tests
+
+# Copyright (c) 2023 Codecoup
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_CONN
+	bool
+	default y
+
+config BT_MAX_CONN
+	int "Maximum number of simultaneous connections (for unit tests)"
+	depends on BT_CONN
+	range 1 250
+	default 1
+	help
+	  Maximum number of simultaneous Bluetooth connections
+	  supported.
+
+config BT_ISO_MAX_CHAN
+	int "Maximum number of simultaneous ISO channels (for unit test)"
+	default BT_MAX_CONN if BT_CONN
+	default 1
+	range 1 64
+	help
+	  Maximum number of simultaneous Bluetooth isochronous channels
+	  supported.at would make inserting a new
+	  segment into a PDU worthwhile.
```

Comment:
Hmm, I've seen such approach. Basically it gives an extra possibility to run the tests with different settings defined in `testcase.yaml` e.g.

```
common:
  tags: test_framework bluetooth
tests:
  bluetooth.audio.ascs.iso_max_chan_1:
    type: unit
    extra_configs:
      - CONFIG_BT_ISO_MAX_CHAN=1
  bluetooth.audio.ascs.iso_max_chan_10:
    type: unit
    extra_configs:
      - CONFIG_BT_ISO_MAX_CHAN=10
```

Reply:
> Basically it gives an extra possibility to run the tests with different settings defined in testcase.yaml e.g.

That can still be done (that's how we test the BT shell).

But redefining the Kconfigs here does not only make it harder to maintain, the Kconfig options you define here may also modify the allowed value range, etc. which could in theory allow you to misconfigure the test, perhaps triggering issues that could otherwise not happen.

Reply:
unittest in tests/bluetooth/controller have been updated recently. 
You should not redefine the Kconfigs, but set them in prj.conf files (see for ex. tests/bluetooth/controller/ctrl_phy_update)

Reply:
@kruithofa OK